website/content/docs/operations/metrics.mdx

```diff
@@ -0,0 +1,66 @@
+  Obtain visibility of various components of a running Boundary
+---
+
+## Boundary Metrics
+
+Boundary provides Prometheus metrics for Controllers and Workers at the ops endpoint on port 9203 through the `/metrics` path. See the [sample code](#sample-code) section for an example listener stanza in a `config.hcl` file.
+
+For more information about Prometheus, see the [Prometheus Overview Guide](https://prometheus.io/docs/introduction/overview). The Prometheus Go library is used to define and export metrics from the Boundary system. The metrics uses the [OpenMetric exposition](https://prometheus.io/docs/instrumenting/exposition_formats/#openmetrics-text-format) format.
+
+## Metrics Collected
+
+The following lists the current metrics used, their descriptions, and full metric names. Metric types collected include latency, traffic, and errors.
+
+### Controller API
+
+| Metric        | Description                                   | Metric name   |
+|---------------|-----------------------------------------------|---------------|
+| Latency       | Historgram of latencies for HTTP requests.    | `boundary_controller_api_http_request_duration_seconds` |
+| Request size  | Histogram of request sizes for HTTP requests  | `boundary_controller_api_http_request_size_bytes`       |
+| Response size | Histogram of response sizes for HTTP requests | `boundary_controller_api_http_response_size_bytes`      |
+
+### Controller Cluster API
+
+| Metric        | Description                                    | Metric name   |
+|---------------|------------------------------------------------|---------------|
+| Latency       | Histogram of latencies for gRPC requests       | `boundary_controller_cluster_http_request_duration_seconds` |
+| Request size  | Histogram of request sizes for gRPC requests   | `boundary_controller_cluster_http_request_size_bytes` |
+| Response size | Histogram of response sizes for gRPC responses | `boundary_controller_cluster_http_response_size_bytes` |
+
+### Worker Proxy Websocket:
+
+| Metric             | Description                                             | Metric name   |
+|--------------------|---------------------------------------------------------|---------------|
+| Active Connections | Count of open websocket connections to Boundary workers | `boundary_worker_proxy_websocket_active_connections`   |
+| Bytes Received     | Count of received bytes for Worker connections          | `boundary_worker_proxy_websocket_received_bytes_total` |
+| Bytes Sent         | Count of sent bytes for Worker connections              | `boundary_worker_proxy_websocket_sent_bytes_total`     |
+
+## Metric Labels
+
+For HTTP requests, the following labels should be used: `path`, `method`, `code`. The value for the `path` label removes any query parameters, replaces unbounded portions of the path to something unbounded, such as resource IDs to "{id}", for example, and maps any paths which are not expected into a single value "unknown."
```

Comment:
The consumer of this documentation will not have any action to take on this.  It is strange to have phrasing like "the following labels should be used" since that is something that we, as engineers in Boundary do when implementing metrics.